src/Test/QuickCheck/Arbitrary.hs

```diff
@@ -165,9 +165,19 @@ import GHC.Generics
 #endif
 
 import qualified Data.Set as Set
-import qualified Data.Map as Map
 import qualified Data.IntSet as IntSet
+#if defined(MIN_VERSION_containers)
+#if MIN_VERSION_containers(0,5,0)
+import qualified Data.Map.Strict as Map
+import qualified Data.IntMap.Strict as IntMap
+#else
+import qualified Data.Map as Map
+import qualified Data.IntMap as IntMap
+#endif
+#else
+import qualified Data.Map as Map
 import qualified Data.IntMap as IntMap
+#endif
```

Comment:
This seems more complicated than it needs to be given that the old code was just straight-up imports?

Reply:
I agree, but since this is in a module that will always be included in the build I think it has to be done with a CPP conditional.

Regarding the `#if defined`, this is the pattern currently being used in `src/Test/QuickCheck/Test.hs` and I assumed I should follow it. However, if we can assume that the build is always going to be done using Cabal, the macro will always be defined so we can drop this extra complexity.

Reply:
I removed the defensive `#if defined`